architecture.html

```diff
@@ -0,0 +1,78 @@
+  <head>
+    <meta charset="UTF-8">
+    <title>Justin Hernandez</title>
+    <link rel="stylesheet" href="css/style.css" media="screen" type="text/css" />
+
+  </head>
+
+  <body>
+    <header>
+        <div class="logo">
+          <h1><a href="#">Justin Hernandez</a></h1>
+        </div> <!-- End Logo -->
+        <div class="main__nav">
+          <ul>
+            <li class="listorder--float" id="showSecondaryNav__Design">
+              <a href="#">Design Work
+              <img src="images/arrow_down.svg" class="arrow"></a>
+            </li>
+
+            <li class="listorder--float hide" id="hideSecondaryNav__Design">
+              <a href="#">Design Work<img src="images/arrow_up.svg" class="arrow"></a>
+            </li>
+
+            <li class="listorder--float" id="showSecondaryNav__Photography">
+              <a href="#">Photography
+              <img src="images/arrow_down.svg" class="arrow"></a>
+            </li>
+
+            <li class="listorder--float hide" id="hideSecondaryNav__Photography">
+              <a href="#">Photography<img src="images/arrow_up.svg" class="arrow"></a>
+            </li>
+
+            <li class="listorder--float"><a href="#">Personal Project</a></li>
+            <li class="listorder--float"><a href="#">About Me</a></li>
+
+          </ul>
+
+        </div> <!-- End Main Nav -->
+        <div class="socialmedia">
+          <ul>
+            <li class="listorder--float"><a href="#"><img src="http//placehold.it/75x75"></a></li>
+            <li class="listorder--float"><a href="#"><img src="http//placehold.it/75x75"></a></li>
+            <li class="listorder--float"><a href="#"><img src="http//placehold.it/75x75"></a></li>
+          </ul>
+        </div> <!-- End Social Media -->
+        <div class="clearfix hide" id="secondary__nav--design--container">
+          <div id="secondary__nav--design">
+            <ul>
+              <li class="listorder--float"><a href="dailygreens.html">Daily Greens</a></li>
+              <li class="listorder--float"><a href="#">Texas Junk Fest</a></li>
+              <li class="listorder--float"><a href="#">Anti-Smoking Ad</a></li>
+            </ul>
+          </div><!-- End Secondary Nav Design -->
+        </div> <!-- End Secondary Nav Design Container -->
+
+        <div class="clearfix hide" id="secondary__nav--photography--container">
+          <div id="secondary__nav--photography">
+            <ul>
+              <li class="listorder--float"><a href="architecture.html">Architecture</a></li>
+              <li class="listorder--float"><a href="#">Color Minimal</a></li>
+            </ul>
+          </div><!-- End Secondary Nav Design -->
+        </div> <!-- End Secondary Nav Design Container -->
+
+    </header> <!-- End Header -->
+
+    <div class="imagegallery clearfix">
+      <img src="http://placehold.it/1920x1080">
+    </div><!-- End Image Gallery -->
+
+
+
+    <script src="http://codepen.io/assets/libs/fullpage/jquery.js"></script>
```

Comment:
Pull in jQuery from Google Hosted Libraries, not CodePen. Google puts time and money into making sure that file is always available, CodePen is a smaller less reliable service.